.circleci/config.yml

```diff
@@ -101,25 +121,45 @@ jobs:
             done
           background: true
 
+      # Run the unit tests themselves
       - run:
           name: run tests
           command: |
-            export DISPLAY=:0
+            export DISPLAY=:0  # we need an X11 display for tests of environment rendering
+            # --cov specifies which directories to report code coverage for
+            # Since we test the installed `seals`, our source files live in `venv`, not in `src/seals`.
+            # --junitxml records test results in JUnit format. CircleCI can parse this to pretty-print results.
```

Comment:
Maybe mention that the directory we chose here corresponds to the `store_test_results` section later.